src/screens/FeedScreen/RenderList.js

```diff
@@ -231,7 +326,8 @@ RenderListFeed.propTypes = {
   source: PropTypes.string,
   hideThreeDot: PropTypes.bool,
   showAnonymousOption: PropTypes.bool,
-  onHeaderOptionClicked: PropTypes.func
+  onDeletePost: PropTypes.func,
+  isSelf: PropTypes.bool
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New props have been added to the PropTypes validation. Ensure that these props are being passed correctly and are of the correct type.
```diff
- onHeaderOptionClicked: PropTypes.func

+ onDeletePost: PropTypes.func,
+ isSelf: PropTypes.bool
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->